src/snackbar/Snackbar.js

```diff
@@ -101,7 +101,7 @@ Snackbar.propTypes = {
     label: PropTypes.string.isRequired,
     onClick: PropTypes.func.isRequired,
   }),
-  text: PropTypes.string.isRequired,
+  text: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```

Comment:
`node` allows `string`, so we can just do:
```suggestion
  text: PropTypes.node.isRequired,
```
😍